com.unity.visualeffectgraph/Editor/Models/VFXGraph.cs

```diff
@@ -38,15 +39,19 @@ static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAsse
                             graph.SanitizeForImport();
                             if (!wasGraphSanitized && graph.sanitized)
                             {
-                                if (assetToReimport == null)
-                                    assetToReimport = new List<string>();
                                 assetToReimport.Add(assetPath);
                             }
                         }
                         catch (Exception exception)
                         {
                             Debug.LogErrorFormat("Exception during sanitization of {0} : {1}", assetPath, exception);
                         }
+
+                        var window = VFXViewWindow.GetWindow(graph, false, false);
```

Comment:
Just a confirmation: can't we have several window opened on the same graph ?

Reply:
No, only one window for a graph. If you try to open it a second time it will only bring the currently opened window in front.